### PR TITLE
Feature/mnist example

### PR DESCRIPTION
**Pull request by VIVelev**
MNIST Training example, made with:
 - Linear layers
 - Sigmoids
 - Softmax
 - CrossEntropy
 - Adam

### CODE REVIEW

**Review by VIVelev — COMMENTED** [review]
LGTM, @ManiacMaxo merge when you approve
